src/pricing/cc-pricing-product-storage.js

```diff
@@ -87,7 +81,7 @@ const SKELETON_TRAFFIC_INTERVALS = [
  * @prop {String} name - Sets the name of the product.
  * @prop {Boolean} noTraffic - Hides the traffic section of the component.
  *
- * @event {CustomEvent<Product>} cc-pricing-product:add-product - Fires the product whenever the "add" button is clicked.
+ * @event {CustomEvent<plan>} cc-pricing-product:add-plan - Fires the plan whenever the "add" button is clicked.
```

Comment:
The `p` of the interface name of the event should be lowercase instead of uppercase
```suggestion
 * @event {CustomEvent<Plan>} cc-pricing-product:add-plan - Fires the plan whenever the "add" button is clicked.
```